samples/drivers/CAN/mcp2515-dts.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019 Karsten Koenig
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&spi1 {
+	status = "ok";
+	cs-gpios = <&gpioa 4 GPIO_DIR_OUT>;
+	mcp2515@0 {
```

Comment:
tried fixing the issue described below by changing this to 
`can1: mcp2515@0 {`
Creates issue of duplicate label for stm32f072b_disco board, lets cmake run successfully for the other boards, but has issues with  fixup `DT_CAN_1_BUS_SPEED` missing during compile step for nucleof091rc and stm32_min_dev board